client/src/common/components/Forms/TextInput.jsx

```diff
@@ -58,6 +58,7 @@ class TextInput extends PureComponent {
         )}
         <input
           className="ss-text-input__input"
+          disabled={!disabled}
```

Comment:
?

Reply:
<img width="1081" alt="Zrzut ekranu 2019-05-01 o 12 34 44" src="https://user-images.githubusercontent.com/27632432/57013819-944d9d80-6c0d-11e9-81b5-b9102ebe0dd0.png">

DIsabled field is controlled by isMember prop.
When user isMember so the prop is set as TRUE - I need to negate this value here to make thisi disabled=false, to keep this field active for him.

When user is not a member and the prop has value 'false' I need to negate this value to true, to keep this field disabled= true.  Cause not member is not able to edit thosee fields.
Same answer for the second comment